pkg/doc/verifiable/presentation.go

```diff
@@ -9,15 +9,14 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-
-	jsonld "github.com/piprate/json-gold/ld"
-	"github.com/xeipuuv/gojsonschema"
-
 	"github.com/hyperledger/aries-framework-go/pkg/doc/jose"
 	docjsonld "github.com/hyperledger/aries-framework-go/pkg/doc/jsonld"
 	"github.com/hyperledger/aries-framework-go/pkg/doc/jwt"
 	"github.com/hyperledger/aries-framework-go/pkg/doc/signature/verifier"
 	jsonutil "github.com/hyperledger/aries-framework-go/pkg/doc/util/json"
+	jsonld "github.com/piprate/json-gold/ld"
+	"github.com/xeipuuv/gojsonschema"
+	"strings"
```

Comment:
do we still have a linter check for grouping imports?
I noticed the linter didn't seem to catch these being in the wrong place.

Reply:
I noticed this too - I remember updating the linter version awhile back - I noticed in other repos that use the newer linter that it's not checking either. Maybe something with the new version